src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/PublicationService.cs

```diff
@@ -119,8 +120,8 @@ private static TopicTree<PublicationTreeNode> BuildTopicTree(Topic topic, IEnume
                 .Include(r => r.Publication)
                 .Where(release => release.PublicationId == publicationId && !release.SoftDeleted)
                 .ToList()
-                .Where(release => IsReleasePublished(release, includedReleaseIds) &&
-                                  IsLatestVersionOfRelease(release.Publication, release.Id))
+                .Where(release => !release.SoftDeleted && IsReleasePublished(release, includedReleaseIds)
```

Comment:
Generally we should never be getting soft deleted entities back in collections.  For this you need to enable that filter by default though.  Look for instances of "HasQueryFilter" in the Admin API code.  We should then probably enable that filter here too against the Release entity, and then you shouldn't see soft-deleted Releases here any more.

Reply:
Nice - hadn't spotted that - cheers!